tests/test_srv6.py

```diff
@@ -627,6 +629,369 @@ def test_srv6(self, dvs, testlog):
         assert nexthop_entries == get_exist_entries(self.adb.db_connection, "ASIC_STATE:SAI_OBJECT_TYPE_NEXT_HOP")
         assert route_entries == get_exist_entries(self.adb.db_connection, "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY")
 
+
+class TestSrv6MySidFpmsyncd(object):
```

Comment:
@cscarpitta Thanks for adding extensive UTs for all the endpoint behaviors. Could you also add a SRV6 steering routes and check for SRV6 nexthop type - SAI_NEXT_HOP_TYPE_SRV6_SIDLIST in ASIC_DB?

Reply:
Please use FPMSyncd to add the route and check the DB entries.

Reply:
Done.

I added two test cases to verify SRv6 steering routes:
* test_AddRemoveSrv6SteeringRouteIpv4
* test_AddRemoveSrv6SteeringRouteIpv6